tools/html_transformer/test/html_transformer_test.js

```diff
@@ -0,0 +1,27 @@
+var expect = require('chai').expect;
+var htmlTransformer = require('../html_transformer');
+
+suite('htmlTransformer', function() {
+  setup(function() {
+    return htmlTransformer.load(__dirname + '/fixtures/basic.html');
+  });
+
+  test('should load html', function() {
+    var doc = htmlTransformer.window.document;
+    var p = doc.querySelector('p');
+    expect(p.textContent).to.equal('A really great thing to say');
+  });
+
+  test('#getSharedDependencies', function() {
+    expect(htmlTransformer.getSharedDependencies())
+      .to
+      .deep
+      .equal([
+        '/shared/style/toolbars.css',
+        'shared/js/component_utils.js',
+        'shared/elements/gaia_subheader/script.js',
+        'shared/style/tabs.css',
+        '/shared/js/gesture_detector.js'
```

Comment:
For this case of ```href="app://theme.gaiamobile.org/shared/elements/gaia-theme/style.css"``` found it at https://github.com/mozilla-b2g/gaia/blob/167a22b2fcba06f9d09472bbaadd877252ef709a/apps/calendar/index.html#L48

It should be included and parse correctly.